src/codemodder/codemodder.py

```diff
@@ -177,15 +149,18 @@ def run(original_args) -> int:
     full_names = [str(path) for path in files_to_analyze]
     logger.debug("Matched files:\n%s", "\n".join(full_names))
 
-    analyze_files(
-        context,
-        files_to_analyze,
-        codemods_to_run,
-        sarif_results,
-        argv,
-    )
+    # run codemods in sequence
+    for codemod in codemods_to_run:
```

Comment:
Could you point to where the libcst tree output of codemod 1 is passed to codemod 2? I expected to see that change in `analyze_files` since that's where we gather the code tree but maybe I"m missing a key thing

Reply:
The parsed tree is not passed to codemod 2. Instead codemod 2 parses the file from disk again. See the comment below for more details.